src/main/java/org/sagebionetworks/bridge/spring/controllers/AssessmentController.java

```diff
@@ -95,7 +96,16 @@ public Assessment getAssessmentByGuid(@PathVariable String guid) {
         }
         return service.getAssessmentByGuid(appId, ownerId, guid);
     }
-    
+
+    @GetMapping("/v1/apps/{appId}/assessments/{guid}")
+    public Assessment getAssessmentByGuidForWorker(@PathVariable String appId, @PathVariable String guid) {
+        getAuthenticatedSession(WORKER);
+        if (SHARED_APP_ID.equals(appId)) {
+            throw new UnauthorizedException(SHARED_ASSESSMENTS_ERROR);
```

Comment:
So... the way things currently work, only shared assessments can be used by all organizations so the ARC assessments are all "shared". Won't those still need to be scored and have a CVS file created by the worker?

Reply:
The API doesn't allow shared assessments to be consistent with all the other APIs for assessments and shared assessments.

As per stand-up meeting today, the short term hack is to add fallback logic to the Worker to check for local assessments, then fall back to shared assessments. In 2024, we'll need to rework schedules and shared assessments to better fit how we actually use them.